actpool/actpool_test.go

```diff
@@ -1148,6 +1161,47 @@ func (ap *actPool) getPendingBalance(addr string) (*big.Int, error) {
 	return ap.bc.Factory().Balance(addr)
 }
 
+// createState adds a new account with initial balance to the factory
+func createState(sf factory.Factory, cfg config.Config, registry *protocol.Registry, addr string, init *big.Int) (*state.Account, error) {
```

Comment:
Move this function to package "factory" as a static function if we need to declare it multiple times, and name it as "createTestAccount"

Reply:
fixed in latest commit